diff --git a/node_modules/@ethersproject/providers/lib.esm/ankr-provider.d.ts b/node_modules/@ethersproject/providers/lib.esm/ankr-provider.d.ts
index 5aa8a46..71a3ec5 100644
--- a/node_modules/@ethersproject/providers/lib.esm/ankr-provider.d.ts
+++ b/node_modules/@ethersproject/providers/lib.esm/ankr-provider.d.ts
@@ -1,6 +1,6 @@
 import { Network } from "@ethersproject/networks";
 import { UrlJsonRpcProvider } from "./url-json-rpc-provider";
-import type { ConnectionInfo } from "@ethersproject/web";
+import { ConnectionInfo } from "@ethersproject/web";
 export declare class AnkrProvider extends UrlJsonRpcProvider {
     readonly apiKey: string;
     isCommunityResource(): boolean;
diff --git a/node_modules/@ethersproject/providers/lib/ankr-provider.d.ts b/node_modules/@ethersproject/providers/lib/ankr-provider.d.ts
index 5aa8a46..71a3ec5 100644
--- a/node_modules/@ethersproject/providers/lib/ankr-provider.d.ts
+++ b/node_modules/@ethersproject/providers/lib/ankr-provider.d.ts
@@ -1,6 +1,6 @@
 import { Network } from "@ethersproject/networks";
 import { UrlJsonRpcProvider } from "./url-json-rpc-provider";
-import type { ConnectionInfo } from "@ethersproject/web";
+import { ConnectionInfo } from "@ethersproject/web";
 export declare class AnkrProvider extends UrlJsonRpcProvider {
     readonly apiKey: string;
     isCommunityResource(): boolean;
diff --git a/node_modules/@ethersproject/providers/src.ts/ankr-provider.ts b/node_modules/@ethersproject/providers/src.ts/ankr-provider.ts
index 844322f..92b1655 100644
--- a/node_modules/@ethersproject/providers/src.ts/ankr-provider.ts
+++ b/node_modules/@ethersproject/providers/src.ts/ankr-provider.ts
@@ -4,7 +4,7 @@ import { Network } from "@ethersproject/networks";
 import { showThrottleMessage } from "./formatter";
 import { UrlJsonRpcProvider } from "./url-json-rpc-provider";
 
-import type { ConnectionInfo } from "@ethersproject/web";
+import { ConnectionInfo } from "@ethersproject/web";
 
 import { Logger } from "@ethersproject/logger";
 import { version } from "./_version";
