diff --git a/node_modules/ethers/lib.esm/ethers.d.ts b/node_modules/ethers/lib.esm/ethers.d.ts
index 52d8e69..fdb3ed8 100644
--- a/node_modules/ethers/lib.esm/ethers.d.ts
+++ b/node_modules/ethers/lib.esm/ethers.d.ts
@@ -8,7 +8,7 @@ import { getDefaultProvider } from "@ethersproject/providers";
 import { Wordlist, wordlists } from "@ethersproject/wordlists";
 import * as utils from "./utils";
 import { ErrorCode as errors } from "@ethersproject/logger";
-import type { TypedDataDomain, TypedDataField } from "@ethersproject/abstract-signer";
+import { TypedDataDomain, TypedDataField } from "@ethersproject/abstract-signer";
 import { BigNumberish } from "@ethersproject/bignumber";
 import { Bytes, BytesLike, Signature } from "@ethersproject/bytes";
 import { Transaction, UnsignedTransaction } from "@ethersproject/transactions";
diff --git a/node_modules/ethers/lib/ethers.d.ts b/node_modules/ethers/lib/ethers.d.ts
index 52d8e69..fdb3ed8 100644
--- a/node_modules/ethers/lib/ethers.d.ts
+++ b/node_modules/ethers/lib/ethers.d.ts
@@ -8,7 +8,7 @@ import { getDefaultProvider } from "@ethersproject/providers";
 import { Wordlist, wordlists } from "@ethersproject/wordlists";
 import * as utils from "./utils";
 import { ErrorCode as errors } from "@ethersproject/logger";
-import type { TypedDataDomain, TypedDataField } from "@ethersproject/abstract-signer";
+import { TypedDataDomain, TypedDataField } from "@ethersproject/abstract-signer";
 import { BigNumberish } from "@ethersproject/bignumber";
 import { Bytes, BytesLike, Signature } from "@ethersproject/bytes";
 import { Transaction, UnsignedTransaction } from "@ethersproject/transactions";
diff --git a/node_modules/ethers/src.ts/ethers.ts b/node_modules/ethers/src.ts/ethers.ts
index 440af6e..c7fec6d 100644
--- a/node_modules/ethers/src.ts/ethers.ts
+++ b/node_modules/ethers/src.ts/ethers.ts
@@ -21,7 +21,7 @@ import { ErrorCode as errors, Logger } from "@ethersproject/logger";
 ////////////////////////
 // Types
 
-import type { TypedDataDomain, TypedDataField } from "@ethersproject/abstract-signer";
+import { TypedDataDomain, TypedDataField } from "@ethersproject/abstract-signer";
 import { BigNumberish } from "@ethersproject/bignumber";
 import { Bytes, BytesLike, Signature } from "@ethersproject/bytes";
 import { Transaction, UnsignedTransaction } from "@ethersproject/transactions";
